src/docs/manual/02_install.adoc

```diff
@@ -55,66 +53,100 @@ After unzipping, put the dependencies in `resources` folder, so that the structu
 //(New-Object Net.WebClient).DownloadFile('https://github.com/docToolchain/docToolchain/archive/master.zip','master.zip')
 //----
 
-This should already be enough to start a first build:
+== Initialize directory for documents
+
+The next step after getting docToolchain is to initialize a directory where your documents live. In docToolchain this
+directory is named "docDir".
+
+=== Existing documents
 
+If you already have some existing documents in asciidoc format in your project, you need to put the configuration
+file there to inform docToolchain what and how to process. You can do that manually by copying the contents of 
+template_config directory. You can also do that by running `initExisting` task.
 
-.linux with gradle wrapper
+.Linux initExisting example
 [source, bash]
 ----
-./gradlew
+cd <docToolchain home>
+./gradlew -b init.gradle initExisting -PdocDir=<your directory>
 ----
 
-.linux with maven wrapper
+You need to open Config.groovy file and configure names of your files properly.
+You may also change the PDF schema file to your taste.
+
+=== Arc42 from scratch
+
+If you don't have existing documents yet, or if you need a fresh start, you can get the Arc42 template in asciidoc
+format. You can do that by manually downloading from http://arc42.org/download. You can also to that by running
+`initArc42<language>` task. Currently supported languages are:
+
+- DE - German.
+- EN - English.
+- ES - Spanish.
+
+.Linux initArc42EN example
 [source, bash]
 ----
-./mvnw
+cd <docToolchain home>
+./gradlew -b init.gradle initArc42EN -PdocDir=<your directory>
 ----
 
-.Windows with gradle wrapper
+The Config.groovy file is then preconfigured to use the downloaded template.
+
+== Build
+
+This should already be enough to start a first build. Due to some limitations in gradle, you need to run the build from
+the docToolchain home directory, and point docDir property to an absolute directory where your documents live.
```

Comment:
is this really the case? have to check this...

Reply:
I would be happy to avoid the need to cd to docToolchain, I would love to run this tool from the docDir. Let me know if you find any way.